<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultyFlightModelPage</class>
 <widget class="QWidget" name="DifficultyFlightModelPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="SeparateEStart">
     <property name="text">
      <string>Separate engines start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ComplexEManagement">
     <property name="text">
      <string>Complex engine management</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="EngineOverheat">
     <property name="text">
      <string>Engine overheat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="TorqueGyroEffects">
     <property name="text">
      <string>Torque gyro effects</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="FlutterEffect">
     <property name="text">
      <string>Flutter effect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="StallSpins">
     <property name="text">
      <string>Stall spins</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="BlackoutsRedouts">
     <property name="text">
      <string>Blackouts &amp;&amp; redouts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="GLimits">
     <property name="text">
      <string>G-Limits</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="Reliability">
     <property name="text">
      <string>Reliability</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
